### PR TITLE
feat(relayer): relaying packet from Cosmos to Eth

### PR DESCRIPTION
**Pull request by srdtrk**
## Description

closes: #127, #135

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Linked to GitHub issue with discussion and accepted design, OR link to spec that describes this work.
- [x] Wrote unit and integration tests.
- [x] Added relevant natspec and `godoc` comments.
- [x] Provide a [conventional commit message](https://www.conventionalcommits.org/en/v1.0.0/) to follow the repository standards.
- [x] Re-reviewed `Files changed` in the GitHub PR explorer.
- [x] Review `SonarCloud Report` in the comment section below once CI passes.

### CODE REVIEW

File: e2e/interchaintestv8/relayer_test.go [hunk 1]
```diff
@@ -91,3 +104,133 @@ func (s *RelayerTestSuite) TestRelayerInfo() {
...
+	cosmosUserWallet := s.UserB
+	cosmosUserAddress := cosmosUserWallet.FormattedAddress()
+	sendMemo := "nonnativesend"
+
+	var (
+		transferCoin sdk.Coin
+		txHash       []byte
+	)
+	s.Require().True(s.Run("Send transfer on Cosmos chain", func() {
+		// We need the timeout to be a whole number of seconds to be received by eth
```
Comment (gjermundgaraba) [inline e2e/interchaintestv8/relayer_test.go:127]: Nit: I think maybe this comment comes from the other tests and is a relic from when the timeouts were not the same. I think we can just delete this comment everywhere as it is more confusing than clarifying at this point :D

File: e2e/interchaintestv8/relayer_test.go [hunk 2]
```diff
@@ -91,3 +104,133 @@ func (s *RelayerTestSuite) TestRelayerInfo() {
...
+				Denom:   transferCoin.Denom,
+			})
+			s.Require().NoError(err)
+			s.Require().NotNil(resp.Balance)
+			s.Require().Equal(testvalues.InitialBalance-testvalues.TransferAmount, resp.Balance.Amount.Int64())
+		}))
+	}))
+
+	var multicallTx []byte
+	s.Require().True(s.Run("Retrieve relay tx to Ethereum", func() {
```
Comment (gjermundgaraba) [inline e2e/interchaintestv8/relayer_test.go:177]: <3

File: e2e/interchaintestv8/relayer_test.go [hunk 3]
```diff
@@ -91,3 +104,133 @@ func (s *RelayerTestSuite) TestRelayerInfo() {
...
+		resp, err := s.RelayerClient.RelayByTx(context.Background(), &relayertypes.RelayByTxRequest{
+			SourceTxIds:     [][]byte{txHash},
+			TargetChannelId: s.TendermintLightClientID,
+		})
+		s.Require().NoError(err)
+		s.Require().NotEmpty(resp.Tx)
+		s.Require().Equal(resp.Address, ics26Address.String())
+
+		multicallTx = resp.Tx
```
Comment (gjermundgaraba) [inline e2e/interchaintestv8/relayer_test.go:186]: Would we want it to be a multi call tx when it's a single tx? I haven't measured, but I would assume the cost is higher than a direct tx?

**Review by gjermundgaraba — APPROVED** [review]
LGTM! 🚀

File: e2e/interchaintestv8/relayer_test.go (hunk 3 above)
Comment (srdtrk) [reply e2e/interchaintestv8/relayer_test.go:186]: I think consistency is always good. This is not a usecase we are optimizing for, and the gas cost of using multicall vs standard entry point should be very low, compared to all the other reasons gas will be high in that scenario

File: e2e/interchaintestv8/relayer_test.go (hunk 1 above)
Comment (srdtrk) [reply e2e/interchaintestv8/relayer_test.go]: done

**Pull request merged**